### DISCUSSION TITLE
I entered accounts into ente auth.  And I thought I saved my password and key on Password saver.  But unfortunately, it did not save the password or key.  Can you help me to login to my acct?

### DISCUSSION THREAD

**Discussion by seedhay** [ente-io/ente#3648] [Q&A]
I entered accounts into ente auth.  And I thought I saved my password and key on Password saver.  But unfortunately, it did not save the password or key.  Can you help me to login to my acct?  What steps can I perform to gain access to Ente Auth.  I only have my email.  Freaking out.

**Comment by Damariobros** [comment]
Unfortunately there is no way to access your account without your password or your recovery key. Your account is encrypted using them, and decrypting your account requires one of them. Unless you're from the future and have a quantum computer capable of breaking encryption, I'm sorry to say but it is physically impossible to access your account.

Next time, when setting your password, it may be a better option to first save the password to the password manager, and THEN use the password manager to autofill the 'new password' and 'confirm password' fields. It can't autofill without first saving the password.

**Comment by seedhay** [comment]
You sent me exactly the reply I thiought. Ente Auth for Zorin is way too buggy anyways.
Found a better more stable auth app. Thanks for nothing.

Sent with [Proton Mail](https://proton.me/mail/home) secure email.
